### DISCUSSION TITLE
An unexpected internal error occurred

### DISCUSSION THREAD

**Discussion by d1d4r** [vercel/community#4667] [Help] [answered]
### Summary

hi, i'm from iraq and i wanna create vercel account i fill input with my number and country code
 +964 750 xxx xxxx but vercel show this error message (An unexpected internal error occurred)

### Steps to Reproduce

An unexpected internal error occurred

**Comment by swarnava** [comment, marked as answer]
Please email us at registration@vercel.com from your registered email address.

**Reply by d1d4r** [reply]
Please check email
